helix-core/src/main/java/org/apache/helix/task/JobRebalancer.java

```diff
@@ -57,6 +57,7 @@
       new FixedTargetTaskAssignmentCalculator();
   private static TaskAssignmentCalculator genericTaskAssignmentCal =
       new GenericTaskAssignmentCalculator();
+  private static Map<String, Integer> participantActiveTaskCount = new HashMap<String, Integer>();
```

Comment:
This variable should start with `_`. I believe `genericTaskAssignmentCal` above was wrong.